### DISCUSSION TITLE
No PostgREST OpenAPI exposed

### DISCUSSION THREAD

**Discussion by janwilmake** [supabase/supabase#28526] [Feature Requests] [answered]
At https://supabase.com/docs/guides/api you state the following:

> Supabase provides a RESTful API using [PostgREST](https://postgrest.org/). This is a very thin API layer on top of Postgres. It exposes everything you need from a CRUD API at the URL https://<project_ref>.supabase.co/rest/v1/. The REST interface is automatically reflected from your database's schema and is:

On https://docs.postgrest.org/en/v12/references/api/openapi.html I found that the root endpoint should expose the OpenAPI Swagger v2 spec

However, this doesn't happen when I curl `curl -v GET "https://MYPROJECT.supabase.co/rest/v1"`

Have you disabled this on purpose? Is there a way to access this OpenAPI spec? It would be very helpful, as I'm trying to build something in a language you don't have an SDK for.

**Comment by GaryAustin1** [comment, marked as answer]
Seems to be working, but you need a trailing / on the url.
![image](https://github.com/user-attachments/assets/bbbd65b0-b915-427d-936f-176e536a1e5f)

**Comment by janwilmake** [comment]
Oh wow, thanks!

**Comment by sebastianrothbucher** [comment]
indeed - thanks
